packages/react/src/collapsible/panel/useCollapsiblePanel.ts

```diff
@@ -132,9 +133,9 @@ export function useCollapsiblePanel(
     let frame = -1;
     let nextFrame = -1;
 
-    frame = requestAnimationFrame(() => {
+    frame = AnimationFrame.scheduler.request(() => {
```

Comment:
It might be more convenient to exporse the scheduler methods directly on AnimationFrame (so we can call `AnimationFrame.schedule()`). WDYT?

Reply:
I did think about it, but I went with this because it keeps the implementation for `AnimationFrame` and `Scheduler` in two separate classes, which makes it easier to read & understand the boundary. Both classes also have `request` and `cancel` methods, so I didn't like the name collision only differentiated with `static`. But I could be convinced to mash it all in one class.

I wouldn't call it `AnimationFrame.schedule()` though, I like keeping the API names as they are.

Reply:
I could also just add static methods on `AnimationFrame` that carry the call to scheduler, that would keep the classes separate.

Reply:
I've added `static` methods on `AnimationFrame` that forward the calls to `Scheduler`, looks like a good option for readability internally & externally.